src/Concerns/ManagesUsageBilling.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Laravel\Cashier\Concerns;
+
+use Illuminate\Support\Collection;
+use Stripe\Billing\MeterEvent;
+
+trait ManagesUsageBilling
+{
+    /**
+     * Get all the defined billing meters.
+     *
+     * @param  array  $options
+     * @param  array  $requestOptions
+     * @return \Illuminate\Support\Collection
+     */
+    public function meters(array $options = [], array $requestOptions = []): Collection
```

Comment:
If we end up supporting Enums for meter names / IDs below, should we also support them here?